kubetest2-tf/data/k8s-ansible/patch-nodes.yaml

```diff
@@ -12,4 +12,11 @@
   serial: 1
   become: yes
   roles:
-    - role: reboot-sequentially
+    - reboot-sequentially
+
+- name: Set desired SMT levels on nodes
```

Comment:
Necessary as the previous SMT configurations are not persisting on a node restart.